SPECS/pg_bulkload-pg15.spec

```diff
@@ -0,0 +1,104 @@
+# SPEC file for pg_bulkload on PostgreSQL 15
+# Copyright (C) 2022-2023 NIPPON TELEGRAPH AND TELEPHONE CORPORATION
+
+%define sname                   pg_bulkload
+%define pgmajorversion  15
+
+%define _prefix                 /usr/pgsql-%{pgmajorversion}
+%define _libdir                 %{_prefix}/lib
+%define _bcdir			%{_libdir}/bitcode/pg_bulkload
+
+Summary:        High speed data load utility for PostgreSQL
+Name:           %{sname}
+Version:        3.1.20
+Release:        1%{?dist}
+License:        BSD
+Group:          Applications/Databases
+# You can get the tarball by following: https://github.com/ossc-db/pg_bulkload/archive/%{version}.tar.gz
+Source0:        %{sname}-%{version}.tar.gz
+URL:            http://ossc-db.github.io/pg_bulkload/index.html
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-%(%{__id_u} -n)
+
+BuildRequires:  postgresql15-devel, postgresql15
+Requires:       postgresql15
+
+
+%description
+pg_bulkload provides high-speed data loading capability to PostgreSQL users.
+
+When we load huge amount of data to a database, it is common situation that data set to be loaded is valid and consistent. For example, dedicated tools are used to prepare such data, providing data validation in advance. In such cases, we'd like to bypass any overheads within database system to load data as quickly as possible. pg_bulkload is developed to help such situations. Therefore, it is not pg_bulkload's goal to provide detailed data validation. Rather, pg_bulkload asumes that loaded data set is validated by separate means. If you're not in such situation, you should use COPY command in PostgreSQL.
+
+
+%package llvmjit
+Requires: postgresql14-server, postgresql14-llvmjit
```

Comment:
15 is correct?

Reply:
Sorry, 15 is correct. I will chang the package name.